### DISCUSSION TITLE
Is Scheduled job possible with DaemonJob?

### DISCUSSION THREAD

**Discussion by farahfa** [metacontroller/metacontroller#205] [Q&A]
I want to run an app once a month using daemonjob (as it is required to run on ALL nodes), however I don't see anything about this, so is this possible? If so, is there an example of this? And if it's not possible, is there a work-around to this (even if it's hacky)?

**Comment by grzesuav** [comment]
Hi, metacontroller does not have any notion of periodic job, so it would be implemented on controller side

**Comment by grzesuav** [comment]
linking to slack discussion https://kubernetes.slack.com/archives/CA0SUPUDP/p1614293745006800

**Reply by grzesuav** [reply]
to summarize, currently metacontroller  does not have notion of `resync` in a cron manner. However it clear caches every `30` minutes. So your operator can produce a `mark` (arbitrary resource with some timestamp) and on `resync` just check if it's the timeto trigger deamonjob
